File: productmodule/src/main/java/com/example/productmodule/entity/Comment.java
package com.example.productmodule.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;
import java.io.Serializable;
import java.time.LocalDate;

@Data
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "comments", schema = "public")
public class Comment implements Serializable {
    private static final long serialVersionUID = 1L;

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "name_cli")
    private String nameCli;

    @Column(name = "date")
    private LocalDate date;

    @Column(name = "comment")
    private String comment;

    @Column(name = "rate")
    private Double rate;

    @Column(name = "id_pro")
    private Long idPro;
}

File: frontendmodule/src/app/entity/comment.ts
export interface IComment {
  id?: any;
  nameCli?: string;
  date?: any;
  comment?: any;
  rate?: any;
  idPro?: any;
}

export class Comment implements IComment {
  constructor(
    public id?: any,
    public nameCli?: string,
    public date?: any,
    public comment?: any,
    public rate?: any,
    public idPro?: any
  ) {
  }
}
